vllm/v1/worker/gpu_worker.py

```diff
@@ -128,13 +128,22 @@ def init_device(self):
             gc.collect()
             torch.cuda.empty_cache()
             self.init_gpu_memory = torch.cuda.mem_get_info()[0]
+            backend = "nccl"
+        elif self.device_config.device.type == "xpu" and current_platform.is_xpu(
+        ):
+            self.device = torch.device(f"xpu:{self.local_rank}")
+            torch.xpu.set_device(self.device)
+            torch.xpu.empty_cache()
+            self.init_gpu_memory = torch.xpu.get_device_properties(
+                self.local_rank).total_memory
+            backend = "ccl"
```

Comment:
Not sure if we can also come up with similiar way to hide these part as well, I think maybe we can abstract that to xpu.py?